oak_containers_hello_world_trusted_app/proto/interface.proto

```diff
@@ -18,12 +18,14 @@ syntax = "proto3";
 
 package oak.containers.example;
 
+import "oak_crypto/proto/v1/crypto.proto";
+
 message HelloRequest {
-  string name = 1;
+  oak.crypto.v1.EncryptedRequest encrypted_request = 1;
```

Comment:
I would have expected this to be the other way around: that the encrypted request's payload is a HelloRequest and the service signature would change to `rpc EncryptedHello(EncryptedRequest) returns (EncryptedResponse) {}`.

Having said that, I think it is fine to leave as is for now and change once we have a more complex Oak Containers example.

Reply:
I think it will be more convenient to application developers to have their own application specific messages, and use explicit `EncryptedRequest` and `EncryptedResponse` be used inside those application specific messages instead of `bytes encrypted_request`.